src/styles/components/Overlay.scss

```diff
@@ -33,6 +33,24 @@
     line-height: 2.125rem;
     letter-spacing: -5;
 
+    > p {
+      @include media("md-up") {
```

Comment:
This crops the first line of a paragraph's line height (why the email sign up was not perfectly aligned with the top of the paragraph).